ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/eventhandler/MetricsFragmentEventListener.java

```diff
@@ -1,17 +1,22 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.eventhandler;
 
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.events.fragmentation.FragmentDeletedEvent;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.events.fragmentation.BulkFragmentDeletedEvent;
 import io.micrometer.core.instrument.Metrics;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
 
 @Component
 public class MetricsFragmentEventListener {
 
-    private static final String LDES_SERVER_DELETED_FRAGMENTS_COUNT = "ldes_server_deleted_fragments_count";
-    @EventListener
-    public void handleFragmentDeletedEvent(FragmentDeletedEvent event) {
-        String viewName = event.ldesFragmentIdentifier().getViewName().asString();
-        Metrics.counter(LDES_SERVER_DELETED_FRAGMENTS_COUNT, "view",  viewName, "fragmentation-strategy", "pagination").increment();
-    }
+	private static final String LDES_SERVER_DELETED_FRAGMENTS_COUNT = "ldes_server_deleted_fragments_count";
+	private static final String VIEW = "view";
+	private static final String FRAGMENTATION_STRATEGY = "fragmentation-strategy";
+
+	@EventListener
+	public void handleBulkFragmentDeletedEvent(BulkFragmentDeletedEvent event) {
+		event.ldesFragmentIdentifiers().forEach(ldesFragmentIdentifier -> {
```

Comment:
Why don't we increment in bulk? -> ...increment(event.ldesFragmentIdentifiers().size())?

Reply:
well we handle counters for individual views, so i can do a map reduce kind of thing, but idk if it will be so much better than this